appcheck/firebase-appcheck/ktx/ktx.gradle

```diff
@@ -27,9 +27,9 @@ firebaseLibrary {
 
 android {
     namespace "com.google.firebase.appcheck.ktx"    
-    compileSdkVersion project.targetSdkVersion
+    compileSdkVersion project.compileSdkVersion
     defaultConfig {
-        minSdkVersion 16
+        minSdkVersion project.minSdkVersion
```

Comment:
You are also going to need to change the versions in the manifests, like for example https://github.com/firebase/firebase-android-sdk/blob/ep/min-sdk/appcheck/firebase-appcheck/src/main/AndroidManifest.xml

Reply:
I decided to go through and change all the manifests, which was a lot of them, most of which were commented out and seemingly pinned to ancient versions

Reply:
Yes, thanks for updating them. They are meant to be commented out.